docs/releases/versions/version-v9.0.0.md

```diff
@@ -61,7 +61,7 @@ multiple members to a community. Members can be either people or groups:
 ![Community members](img/../v9.0/community-members.png)
 
 It's possible to disable support for group members via the ``COMMUNITIES_GROUPS_ENABLED``
-feature flag.
+feature flag. See [Groups integration](https://inveniordm.docs.cern.ch/customize/authentication/#groups).
```

Comment:
I would mention this flag also above, where you added the groups feature description, otherwise the users don't know they can disable/enable it without reading release notes